docs/developer/architecture/logs.rst

```diff
@@ -0,0 +1,90 @@
+Log Centralisation
+==================
+
+Context
+-------
+
+MetalK8s value is to provide, out of the box, some services in order to ease
+the monitoring and operation of the platform as well as workload running on top
+of it.
+
+MetalK8s provides monitoring service with Prometheus and AlertManager in
+order to expose system performance time series and alerts.
+
+The logs generated by the platform and the workloads constitute an essential
+piece of information that can be very helpful when it comes to understand the
+root cause of a failure or a performance degradation. Because of the
+distributed nature of MetalK8s and workloads running of top of it, it is
+important to have some tooling to collect and aggregate all logs in order to
+make them browsable from a central endpoint. It is also important to ease
+browsing and correlation with other type of indicators such as KPIs and alerts.
```

Comment:
Can't we use labels? and have shared labels between Prometheus KPIs and Logs? For example if we have some label for PUT Object OPS metric, can't we have the same label for logs about Object Put (from cloudserver, or/and other components)? If we ca correlate alert, kpis and logs within a single dashboard it will drastically speed up troubleshooting.

Now Loki is not indexing the content of the logs but only offer the ability to label the different log streams.

Reply:
Right, we have arbitrary labels, so indeed, we can use them to provide correlation. Good point.